src/main/java/com/icthh/xm/uaa/service/query/UserQueryService.java

```diff
@@ -71,14 +81,16 @@ private Specification<User> createSoftSpecification(SoftUserFilterQuery query) {
     }
 
     private Stream<Optional<Specification<User>>> createStrictSpecs(StrictUserFilterQuery filterQuery) {
-        return Stream.of(
+        Stream<Optional<Specification<User>>> filters = Stream.of(
```

Comment:
It is better to use optional.stream() to avoid Stream<Optional<Spec<User>>> and concat Stream<Spec<user>> without optional wrapper

Reply:
This is just extention of existing method. Usage Stream<Optional<Spec>> already presents before my changes.